content/sensu-core/2.0/reference/backend.md

```diff
@@ -0,0 +1,585 @@
+
+# /etc/sensu/backend.yml example
+deregistration-handler: "/path/to/handler.sh"{{< /highlight >}}
+
+
+| log-level  |      |
+-------------|------
+description  | Logging level: `panic`, `fatal`, `error`, `warn`, `info`, or `debug`
+type         | String
+default      | `"warn"`
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --log-level debug
+
+# /etc/sensu/backend.yml example
+log-level: "debug"{{< /highlight >}}
+
+
+| state-dir  |      |
+-------------|------
+description  | Path to Sensu state storage 
+type         | String
+default      | <ul><li>Windows: `C:\\ProgramData\sensu\data`</li><li>FreeBSD: `/usr/local/etc/sensu/backend.yml`</li><li>Linux: `/var/lib/sensu`</li></ul>
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --state-dir /var/lib/sensu
+sensu-backend start -d /var/lib/sensu
+
+# /etc/sensu/backend.yml example
+state-dir: "/var/lib/sensu"{{< /highlight >}}
+
+
+### Agent communication configuration flags
+
+| agent-host   |      |
+---------------|------
+description    | agent listener host, listens on all IPv4 and IPv6 addresses by default
+type           | String
+default        | `"[::]"`
+example        | {{< highlight shell >}}# Command line example
+sensu-backend start --agent-host 127.0.0.1
+
+# /etc/sensu/backend.yml example
+agent-host: "127.0.0.1"{{< /highlight >}}
+
+
+| agent-port |      |
+-------------|------
+description  | agent listener port
+type         | Integer
+default      | `8081`
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --agent-port 8081
+
+# /etc/sensu/backend.yml example
+agent-port: 8081{{< /highlight >}}
+
+### Security configuration flags
+
+| cert-file  |      |
+-------------|------
+description  | SSL/TLS certificate
+type         | String
+default      | `""`
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --cert-file /path/to/ssl/cert.pem
+
+# /etc/sensu/backend.yml example
+cert-file: "/path/to/ssl/cert.pem"{{< /highlight >}}
+
+
+| key-file   |      |
+-------------|------
+description  | SSL/TLS certificate key
+type         | String
+default      | `""`
+example      | {{< highlight shell >}}# Command line example
+sensu-backend start --key-file /path/to/ssl/key.pem
+
+# /etc/sensu/backend.yml example
+key-file: "/path/to/ssl/key.pem"{{< /highlight >}}
+
+
+| trusted-ca-file |      |
+------------------|------
+description       | SSL/TLS certificate authority
+type              | String
+default           | `""`
+example           | {{< highlight shell >}}# Command line example
+sensu-backend start --trusted-ca-file /path/to/trusted-certificate-authorities.pem
+
+# /etc/sensu/backend.yml example
+trusted-ca-file: "/path/to/trusted-certificate-authorities.pem"{{< /highlight >}}
+
+
+| insecure-skip-tls-verify |      |
+---------------------------|------
+description                | skip ssl verification
```

Comment:
The name says it all, but it might be nice to mention here that this shouldn't be used in production.